spec/functions/generate_password_spec.rb

```diff
@@ -0,0 +1,12 @@
+require 'spec_helper'
```

Comment:
We could remove generate_password as well. We make use of the foreman modules version of this idea.